files.sh

```diff
@@ -2,7 +2,7 @@
 
 fn nash_complete_paths(parts, line, pos) {
 	partsz   <= len($parts)
-	last     <= -expr $partsz - 1
+	last     <= -echo $partsz - 1 | bc
```

Comment:
@tiago4orion I think the points I raised where all improvements. If you want you can merge the multiple assigments, since it is stable, and create a new branch to implement the improvements